### DISCUSSION TITLE
Can't compile APK with Distriqt and 33.1.1.856

### DISCUSSION THREAD

**Discussion by yvant** [airsdk/Adobe-Runtime-Support#1940] [General]
Hi, 

I've been trying to build and APK with FlashDevelop on windows, java 11.0.15.1.
I've used manual application.xml and APM as well, and I have that message:
`error 402: Current Android sdk version doesn't support intent as xml tag`

targetSDK is 30, minSDK is 19
I've set ANDROID_SDK= in setupSDK.bat and on adt.cfg

Am I missing something?

Thanks!

**Comment by marchbold** [comment]
This was a small bug in the latest air SDK, there's a fix available here:

https://github.com/airsdk/Adobe-Runtime-Support/issues/1905

**Comment by yvant** [comment]
Thanks for the help, I missed that!
